main.go

```diff
@@ -88,32 +90,17 @@ var (
 		RunE:   deleteDb,
 	}
 
-	syncCmd = &cobra.Command{
-		Use:   "sync [@DB]",
-		Short: "Sync local db with latest Charm Cloud db.",
-		Args:  cobra.MaximumNArgs(1),
-		RunE:  sync,
-	}
-
-	resetCmd = &cobra.Command{
-		Use:   "reset [@DB]",
-		Short: "Delete local db and pull down fresh copy from Charm Cloud.",
-		Args:  cobra.MaximumNArgs(1),
-		RunE:  reset,
-	}
-
```

Comment:
I think we should leave this and add a warning in the long description that the service will go down soon. This give people some time to pull their data using `sync`

Reply:
they can use the previous version for that... same thing for glow...